vid/src/advz.rs

```diff
@@ -689,6 +753,12 @@ where
                 })
                 .collect::<Result<Vec<_>, VidError>>()?;
 
+            // distribute evals from each poly among the storage nodes
+            //
+            // TODO perf: runtime is O(num_polys * payload_size).
```

Comment:
> Not sure about the goal of this comment. Building this matrix doesn't seem like a bottleneck.

The goal is to flag a quadratic dependence on payload size. It might not be a perf bottleneck now (which is why I'm not doing anything about it) but I like to flag any code snippet with bad asymptotics to ease life for future maintainers.

Comment revised in https://github.com/EspressoSystems/jellyfish/pull/674/commits/f3961714aecf7a126aff7a8aa155a6d5fa4feb81 to
```rust
// perf warning: runtime is quadratic in payload_size
```

> Also, isn't `num_polys` complexity O ( payload _ size multiplicity ∗ recovery _ threshold )

Correct. I'm trying optimize for readability.

Reply:
My bad. This code is actually only linear in the erasure-coded payload size. (It iterates through each polynomial evaluation exactly once.) I'll remove this comment in a future PR.